dashboard/app/models/user.rb

```diff
@@ -2114,7 +2114,7 @@ def clear_user_and_mark_purged
 
     self.purged_at = Time.zone.now
 
-    save!
+    save!(validate: false)
```

Comment:
I'm not a big fan of adding `validate: false` if we can avoid it.

Personally I would prefer to replace the validation with `validate :complete_school_info, if: :school_info_id_changed?, unless: Proc.new { |u| u.purged_at.present? }`. Happy to be overruled if others think differently though

Reply:
@bethanyaconnor I like how your recommended change reads!!

I am curious as to why you are not a fan of validate: false?

Reply:
It would override *all* validations, so if we have any validations for when we can't delete a user, for instance, they wouldn't work any more. Our user model is based off of Devise models so it's hard to know for sure what's going on under the hood and, to me, the code I suggested (or something similar) more precisely describes what we want to do (validate school info when school info changes and it's not a purged user).

Reply:
Don't forget to remove the `validate: false` if you're going with the `unless` above!

Reply:
@bethanyaconnor Edit complete.  Thanks!